tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -188,6 +188,13 @@ Verify Favorite Resource Cards
     Favorite Items Should Be Listed First When Sorted By    ${favorite_ids}    duration
     [Teardown]    Remove Items From favorites    @{favorite_ids}
 
+Verify Dashboard Pod Is Not Getting Restarted	
```

Comment:
The test in Polarion have these steps:
1. Look odh Dashboard Pod
2. Check the odh dashboard is not restarted by the internal pilot by the memory limit 

Right now, how the test is implemented, I see it more as a post-install check (it could be renamed to `Verify Dashboard Pod Is Not Getting Restarted During Installation`)

@pablofelix @bdattoma , I think the test in polarion is not very clear. What do you mean with the internal pilot? Is this meant to be a Sandbox test?

Reply:
the test should check that the dashboard pods don't have a Out-of-Memory error, because this is why the TC was created. 
Already talked yday with @rkamble5 and he will try to reproduce the OOM error on a cluster to refine this implementation